public/friendlycode/js/fc/ui/editor-panes.js

```diff
@@ -2,65 +2,27 @@ define(function(require) {
   var $ = require("jquery"),
       Slowparse = require("slowparse/slowparse"),
       ParsingCodeMirror = require("fc/ui/parsing-codemirror"),
-      Help = require("fc/help"),
-      LivePreview = require("fc/ui/live-preview"),
-      ErrorHelp = require("fc/ui/error-help"),
-      ContextSensitiveHelp = require("fc/ui/context-sensitive-help"),
-      Relocator = require("fc/ui/relocator"),
-      HelpMsgTemplate = require("template!help-msg"),
-      ErrorMsgTemplate = require("template!error-msg");
+      Help = require("fc/help");
 
   require('slowparse-errors');
-  require("codemirror/html");
 
   return function EditorPanes(options) {
     var self = {},
         div = options.container,
-        initialValue = options.value || "",
+        initialValue = options.value || "(none)",
```

Comment:
using an actual string, rather than a falsey value, makes it become a magic value if used in any other file (which I believe this value is), so this introduces a potential maintenance bug.